specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2025-11-01-preview/examples/CosmosDBDatabaseAccountCreateMax.json

```
@@ -244,7 +244,9 @@
         },
         "systemData": {
           "createdAt": "2021-03-12T22:05:09Z"
-        }
+        },
+        "throughputPoolDedicatedRUs": 300000,
+        "throughputPoolMaxAchievableRUs": 700000
```

Can we use throughputPoolMaxConsumableRUs in lieu of throughputPoolMaxAchievableRUs and make this change in all relevant places?

```suggestion
        "throughputPoolMaxConsumableRUs": 700000
```